test/miner_poc_SUITE.erl

```diff
@@ -17,6 +17,9 @@
     poc_dist_v5_test/1,
```

Comment:
can you disable the v4 and before tests?  I think that current and one fallback is enough.